libs/python/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+#
+# F E T C H   P Y T H O N   B I N D I N G S   L I B R A R Y
+#
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+project(fetch-python)
+
+# CMake Configuration
+include(${FETCH_ROOT_CMAKE_DIR}/BuildTools.cmake)
+
+# Compiler Configuration
+setup_compiler()
+
+# if (FETCH_ENABLE_PYTHON_BINDINGS)
```

Comment:
commented out if statement is redundant? remove it?

Reply:
looks like FETCH_ENABLE_PYTHON_BINDINGS is still in cmakelists file, why not just uncomment this?

Reply:
```suggestion
```